source/extensions/filters/http/lua/wrappers.cc

```diff
@@ -230,6 +230,12 @@ int StreamInfoWrapper::luaRequestedServerName(lua_State* state) {
   return 1;
 }
 
+int StreamInfoWrapper::luaRouteName(lua_State* state) {
+  const std::string& route_name = stream_info_.getRouteName();
```

Comment:
nit: `absl::string_view` over `const std::string&`

Reply:
`getRouteName()` returns a String. I was trying to use a String View but wasn't sure if it'd have any issues.